source/swap/contracts/Swap.sol

```diff
@@ -8,6 +8,8 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "./interfaces/ISwap.sol";
 
+import "hardhat/console.sol";
```

Comment:
Please remove this and the related lines below @Perelyn-sama

Reply:
On it, sorry about that.